src/Files.App/Views/PaneHolderPage.xaml.cs

```diff
@@ -363,6 +369,28 @@ private void Pane_GotFocus(object sender, RoutedEventArgs e)
 			var activePane = isLeftPane ? PaneLeft : PaneRight;
 			if (ActivePane != activePane)
 				ActivePane = activePane;
+
+			// Add theme shadow to the active pane
+			if (isLeftPane)
+			{
+				PaneRight?.ShellContentThemeShadow.Receivers.Remove(StatusBar);
+				PaneLeft?.ShellContentThemeShadow.Receivers.Add(StatusBar);
```

Comment:
Can we add the receivers to both panes on the initial load instead of on GotFocus?

Reply:
We'll still need to adjust the translation on GotFocus, but this way we can at least optimize the receivers.

Reply:
I suppose yeah.

Reply:
And I’m not sure but should receiver be StatusBar?
I’m wondering if it should be RootContentGrid.
Or is this intended to make sure shadow will be only on the sidebar and be clipped on the left/right/top border?

Reply:
> I’m wondering if it should be [MainPage.RootGrid](https://github.com/files-community/Files/blob/main/src/Files.App/Views/MainPage.xaml#L221).

Let's try RootGrid and see what it looks like.

Reply:
I made the same suggestion, the shadow needs to fall over anywhere in the app window.

Reply:
The truth is, we're not going to see a difference on Windows 11. Let's apply to RootGrid and test on Windows 10.